src/routines/mpilib20_reduce.F90

```diff
@@ -16,68 +17,73 @@ module mpilib20_reduce_m
         module procedure mpilib20_reduce_int_vec
     end interface
     
-    public :: mpilib20_REDUCE
+    public :: mpilib20_reduce
     
 contains
 
-    !> MPI_REDUCE WRAPPER: INTEGER
-    subroutine mpilib20_reduce_int_scalar(sendbuf, recvbuf, operation, mpi_env, root)
-        use mpi_bindings, only : MPI_REDUCE, MPI_Op, &
-            MPI_Datatype, MPI_INTEGER
-        use mpilib20_init_finalise, only : root_id
+    
+    !> Reduces values on all processes to a single value
+    !> on process defined by process_id
+    !> TODO usage examples
+    subroutine mpilib20_reduce_int_scalar(sendbuf, recvbuf, &
```

Comment:
How do we want to line-split (to keep lines below 90 characters(?)). Here I've split after argument 2 because otherwise one goes over 90, but then operation (potentially being removed soon anyway) and mpi_env are grouped. I've put optional arguments on their own line

Reply:
Yeah, not going over 90 (unless a comment) sounds sensible. I often like to put inputs and outputs on different lines but if we retain a consistent arg ordering with MPI commands, then clearly we can't. In which case do whatever looks neat. Not a fan of putting optionals on their own line.

More broadly we should introduce fprettify and set it up how we like, then we can just run it to apply consistent formatting

Reply:
- [ ] 90 characters sounds good. Do what you think looks readable.

Reply:
I'll keep it neat without putting optional arguments on their own line.

Sounds good re fprettify. We should get a config file written for it relatively soon. Once I commit these PR changes I'll have a good and see what get produced.